Fixtures/Miscellaneous/Plugins/DocCForBinaryDependency/Package.swift

```diff
@@ -0,0 +1,25 @@
+// swift-tools-version: 6.0
+// The swift-tools-version declares the minimum version of Swift required to build this package.
+
+import PackageDescription
+
+let package = Package(
+    name: "DemoKit",
+    products: [
+        .library(name: "DemoKit", targets: ["DemoKit"]),
+    ],
+    dependencies: [
+        .package(url: "https://github.com/swiftlang/swift-docc-plugin", from: "1.0.0"),
+    ],
+    targets: [
+        .binaryTarget(
+            name: "MMKV",
+            url: "https://github.com/Kyle-Ye/MMKV/releases/download/v1.3.5-binary/MMKV.xcframework.zip",
```

Comment:
Do we have any official/stable xcframework url from Apple? Or is it acceptable to directly place a plain xcframework binary in the repo?

Use `Kyle-Ye/MMKV` only for placeholder. Will replace it before merge.

Reply:
Update with a plain FooKit.xcframework build locally.